.github/actions/sync-feedback-comment/src/github.ts

```
@@ -12,61 +10,82 @@ export type PullRequestContext = {
   async function getComments() {
-    return await octokit.request(
-      'GET /repos/{owner}/{repo}/issues/{issue_number}/comments',
-      pr_args
-    )
+    try {
+      return await octokit.request(
+        'GET /repos/{owner}/{repo}/issues/{issue_number}/comments',
+        pr_args
+      )
+    } catch (e) {
+      console.log('ERROR: getComments failed, ', e)
+    }
   }
   
   async function createComment(message: string) {
-    return await octokit.request(
-      'POST /repos/{owner}/{repo}/issues/{issue_number}/comments',
-      {
-        ...pr_args,
-        body: message
-      }
-    )
-  }
+    try {
+      return await octokit.request(
+        'POST /repos/{owner}/{repo}/issues/{issue_number}/comments',
+        {
+          ...pr_args,
+          body: message
+        }
+      )
+    } catch (e) {
+      console.log('ERROR: createComment failed, ', e)
+    }
+   }
   
   async function updateComment(
     comment_id: number,
     message: string
   ) {
-    return await octokit.request(
-      'POST /repos/{owner}/{repo}/issues/comments/{comment_id}',
-      {
-        ...pr_args,
-        comment_id,
-        body: message
-      }
-    )
+    try {
+      return await octokit.request(
+        'POST /repos/{owner}/{repo}/issues/comments/{comment_id}',
+        {
+          ...pr_args,
+          comment_id,
+          body: message
+        }
+      )
+    } catch (e) {
+      console.log('ERROR: updateComment failed, ', e)
+    }
   }
   
   async function deleteComment(comment_id: number) {
-    return await octokit.request(
-      'DELETE /repos/{owner}/{repo}/issues/comments/{comment_id}',
-      {
-        ...pr_args,
-        comment_id
-      }
-    )
+    try {
+      return await octokit.request(
+        'DELETE /repos/{owner}/{repo}/issues/comments/{comment_id}',
+        {
+          ...pr_args,
+          comment_id
+        }
+      )
+    } catch (e) {
+      console.log('ERROR: deleteComment failed, ', e)
```

I'd prefer that we also throw the error here, that way it can propagate all the way up to the action caller and surface that something has gone wrong -- rather than continuing through the workflow when we don't expect it to behave correctly anymore